### PR TITLE
[Merged by Bors] - chore(Geometry): remove remaining `open Classical`

### PR DESCRIPTION
**Pull request by grunweg**
---

For two theorems in `PartitionOfUnity`, I'm unsure if this is the best fix, or adding a suitable decidability assumption is better. Feedback on this is welcome!

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Geometry/Manifold/LocalInvariantProperties.lean [hunk 1]
```diff
@@ -42,13 +42,9 @@ coincide on `s`, then `LiftPropWithinAt P g' s x` holds. We can't call it
...
 -/
 
-
 noncomputable section
 
-open scoped Classical
-open Manifold Topology
-
-open Set Filter TopologicalSpace
+open Manifold Topology Set Filter TopologicalSpace
```
Comment (YaelDillies) [inline Mathlib/Geometry/Manifold/LocalInvariantProperties.lean:47]: Same complaint as on the other PR: You're probably mangling `open` and `open_locale`

Comment (grunweg) [reply Mathlib/Geometry/Manifold/LocalInvariantProperties.lean]: Fixed: in fact, the *first* line ought to be *open scoped*.

**Comment by jcommelin** [comment]
LGTM, but is it intentional that you are also deleting a lot of stuff from the nolints file?

bors d+

**Comment by grunweg** [comment]
Not really: this removal is from #15752 (which I mistakenly included... the easiest way forward could be to merge that and then merge master here; that PR should be really easy to review).

**Comment by grunweg** [comment]
Thanks for the review!
bors merge

**Pull request closed without merging**